test/integration/notifications.test.js

```diff
@@ -0,0 +1,217 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const authService = require('../../services/authService')
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+
+const config = require('config')
+const cookieName = config.get('userToken.cookieName')
+
+chai.use(chaiHttp)
+
+describe('Notifications', function () {
+  let jwt
+  beforeEach(async function () {
+    const userId = await addUser()
+    jwt = authService.generateAuthToken({ userId })
+  })
+
+  afterEach(async function () {
+    await cleanDb()
+  })
```

Comment:
We're not getting the notifications from anywhere. Can we use stubbing in this case? Currently we don't have any route which creates notifications. We could create mock of the firestore data and fetch that instead of hitting DB for now. You can refer to #326 for stubbing.

We can later do actual DB calls once we have at least one functions which creates notifications in the DB. For now, we could mock it.